contracts/SpSEIApi3ReaderProxyV1.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.27;
+
+import "@api3/contracts/interfaces/IApi3ReaderProxy.sol";
+import "./interfaces/ISpSEI.sol";
+
+/// @title An immutable proxy contract that reads the spSEI per SEI ratio
```

Comment:
I realized the order of the assets in the text is wrong for the wstETH reader (it suggest we are doing wstETH/stETH). This way it's correct as the price is indeed the spSEI/SEI.

Reply:
You're right that this exchange rate is using the inverse way. That's why the rate is below 1 even though it's a form of a staking contract. I've actually confused myself while reading the comments and fixing them...

The bigger issue though is that this is indeed missing the `InverseApi3ReaderProxyV1` on the exchange rate.